README.asciidoc

```diff
@@ -404,14 +378,15 @@ hook global WinSetOption filetype=<language> %{
 }
 ----
 
-The faces used for semantic tokens and modifiers can be modified in `kak-lsp.toml`, using the `semantic_tokens.faces` array, for example:
+The faces used for semantic tokens and modifiers can be modified by changing the `lsp_semantic_tokens` option, for example:
 
-[source,toml]
+[source,kak]
 ----
-[semantic_tokens]
-faces = [
-    {face="const_variable_declaration", token="variable", modifiers=["constant", "declaration"]},
-]
+set-option global lsp_semantic_tokens %{
+    faces = [
+        {face="const_variable_declaration", token="variable", modifiers=["constant", "declaration"]},
+    ]
+}
```

Comment:
Maybe this snippet should be wrapped with another `filetype`-based hook, to demonstrate and encourage that faces can be matched to what the individual servers support.